operations/firmware/FAS_Use_Cases.md

```diff
@@ -388,7 +388,12 @@ The CMM firmware update process also checks and updates the Cabinet Environmenta
 
     The `--prereq` option ensures all required components are powered on first. The `--continue` option allows the command to complete in systems without fully populated hardware.
 
-6.  After the components have powered on, boot the nodes using the Boot Orchestration Services \(BOS\).
+6.  Bring up the Slingshot Fabric
+    To bring up the Slingshot fabric, see:
```

Comment:
Change to:

Refer to the following documentation for more information on how to bring up the Slingshot Fabric: